### PR TITLE
Job_Sequencing_Problem C_CPP

### PR DESCRIPTION
**Pull request by amanraj-iit**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #issue_number_here 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [
- [X] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [X] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Saviour1001** [comment]
Delete the Qwickshort file as it is not needed. 
image
And add Mark down styling. Here is a link for that. 
[Markdown Styling](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

**Comment by amanraj-iit** [comment]
@Saviour1001 please review

**Comment by Saviour1001** [comment]
Add Markdown styling .... Don't write in plain text, doesn't look good. Use the links provided above

File: C_CPP/Competitive Programming/Job_Sequencing_Problem.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+#Job_Sequencing_Problem
```
Comment (Saviour1001) [inline C_CPP/Competitive Programming/Job_Sequencing_Problem.md:1]:
```suggestion
# Job_Sequencing_Problem
```
Add the spaces after the Hashtags in other places as well

**Comment by amanraj-iit** [comment]
@Saviour1001 please review

**Comment by amanraj-iit** [comment]
@Saviour1001

**Comment by Saviour1001** [comment]
Looks good to me !!

**Comment by amanraj-iit** [comment]
thanks a lot @Saviour1001

**Pull request merged**